crates/pathfinder/src/sequencer.rs

```diff
@@ -395,6 +408,45 @@ impl ClientApi for Client {
             .post_with_json(&req)
             .await
     }
+
+    #[tracing::instrument(skip(self))]
+    async fn add_deploy_account(
+        &self,
+        version: TransactionVersion,
+        max_fee: Fee,
+        signature: Vec<TransactionSignatureElem>,
+        nonce: TransactionNonce,
```

Comment:
This is a bit strange here -- if `nonce` is required as an argument to `add_deploy_account()` then why is it optional in the `DeployAccount` struct?

Reply:
Nice catch, thanks!

Reply:
Done